tsconfig-strict.json

```diff
@@ -194,6 +195,7 @@
     "core/templates/domain/learner_dashboard/non-existent-explorations.model.spec.ts",
     "core/templates/domain/learner_dashboard/non-existent-topics-and-stories.model.ts",
     "core/templates/domain/learner_dashboard/non-existent-topics-and-stories.model.spec.ts",
+    "core/templates/domain/question/QuestionObjectFactory.ts",
```

Comment:
Please add typing to the spec file too

Reply:
> Please add typing to the spec file too
Hello @vojtechjelinek, It's not possible to currently add `QuestionObjectFactorySpec.ts` as it required strict checks to be applied to multiple other files including `core/templates/pages/contributor-dashboard-page/services/translate-text.service.ts` and it's spec file that are currently being worked upon in issue: #10474.